Client/create/create.html

```diff
@@ -5,7 +5,7 @@
     <title>Path Hero</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
-    <!--<link rel="stylesheet" href="http://create.wettowelreactor.com:3000/style/style.css">-->
+    <!-- <link rel="stylesheet" href="http://create.wettowelreactor.com:3000/style/style.css"> -->
```

Comment:
That link is commented out - it only shows as changed because Sublime seems to have added an extra space.